Dockerfile.xcode

```diff
@@ -17,22 +17,25 @@ CMD mkdir -p /root/xcode && \
     xar -xf /root/files/Xcode_10.3.xip && \
     /root/pbzx/pbzx -n Content | cpio -i && \
     cp -r Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk /tmp/MacOSX10.14.sdk && \
-    cp -r Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1 /tmp/MacOSX10.14.sdk/usr/include/c++ && \
+    mkdir -p /tmp/MacOSX10.14.sdk/usr/include/c++ && \
+    cp -r Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include/c++/v1 /tmp/MacOSX10.14.sdk/usr/include/c++/ && \
```

Comment:
Ah the joys of `cp` that uses the target argument either as a target folder to put stuff in, or as a new name for the folder being copied... We lacked the `v1` folder so osxcross would fail.